### DISCUSSION TITLE
Support image clipboard contents. Please.

### DISCUSSION THREAD

**Discussion by lopezpappalardo** [espanso/espanso#2074] [Feature requests and Ideas]
This link talked about my need: https://github.com/espanso/espanso/discussions/1267

I hope you have it on your roadmap. Or is there some workaround?

Thanks.

**Comment by smeech** [comment]
I suspect it's not on the [Roadmap](https://github.com/orgs/espanso/projects/1). We're a small team and unless someone takes an interest and picks it up it won't get done, but thank you for raising the Issue.

As far as a workaround is concerned (and I produce a lot of these, given the above!), if you know the expansion is to handle an image from clipboard one could script saving the clipboard contents to a file, with something like:
```
xclip -selection clipboard -t image/png -o > /tmp/clipboard.png
```
or whatever your OS supports, and then use an `image_path:`, `html:` or `markdown:` replacement to retrieve and inject `/tmp/clipboard.png`. Alternatively, script the injection process with Python or PowerShell etc., and just use Espanso to set it off.
